.github/workflows/coverity.yml

```diff
@@ -50,7 +50,6 @@ jobs:
         -DUMF_DEVELOPER_MODE=OFF
         -DUMF_TESTS_FAIL_ON_SKIP=ON
         -DUMF_BUILD_LIBUMF_POOL_DISJOINT=ON
```

Comment:
please revert this change and disable jemalloc by default here and in other places